x/ibc/20-transfer/keeper/callbacks_test.go

```diff
@@ -1,11 +1,7 @@
 package keeper_test
 
-import (
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	channel "github.com/cosmos/cosmos-sdk/x/ibc/04-channel"
-	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
-)
-
+// TODO: move to 04-channel after CheckOpen implementation in the following PR
```

Comment:
Does there need to be an open issue for this work? Is it blocking the demo or 1.0?

Reply:
blocking 1.0 but not the demo

Reply:
https://github.com/cosmos/cosmos-sdk/issues/5507